src/morphic.js

```diff
@@ -5357,73 +5357,154 @@ CursorMorph.prototype.init = function (aStringOrTextMorph) {
         this.target.setAlignmentToLeft();
     }
     this.gotoSlot(this.slot);
-    this.initializeClipboardHandler();
+    this.initializeTextarea();
 };
 
-CursorMorph.prototype.initializeClipboardHandler = function () {
-    // Add hidden text box for copying and pasting
-    var myself = this,
-        wrrld = this.target.world();
+CursorMorph.prototype.initializeTextarea = function () {
+    var myself = this;
 
-    this.clipboardHandler = document.createElement('textarea');
-    this.clipboardHandler.style.position = 'absolute';
-    this.clipboardHandler.style.top = window.outerHeight;
-    this.clipboardHandler.style.right = '101%'; // placed just out of view
+    this.textarea = document.createElement('textarea');
+    this.textarea.style.zIndex = -1;
+    this.textarea.style.position = 'absolute';
+    this.textarea.wrap = "off";
+    this.textarea.style.overflow = "hidden";
+    this.textarea.style.fontSize = this.target.fontSize + 'px';
```

Comment:
Why does the font size need to be set? If the element is hidden, shouldn't this be unnecessary?

Reply:
Thanks for your feed back.

I included this line of code to align the floating IME window to the position of caret: 

1. Users expect that the IME window appears near the caret (of the text morphic), but the OS/browser will put the IME window near to the caret of underlying textarea.

2. By setting the font size of the text morphic and the textarea to the same, we can minimize the distance of the two carets.

I recall that I added this when I noticed a huge offset of the IME window. I tried to comment off this line to take a screenshot that showcasing the problem, but in my current setting, there is no noticeable difference with/without this code: in both cases, the position of IME window are not perfect but acceptable.